src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -264,6 +264,14 @@
       Condition="'$(SkipVersionGeneration)' != 'true' AND '$(VersionPropsImported)' != 'true'">
     <!-- ############################### -->
 
+    <!-- Get the Url of the Git Repository -->
+    <Exec Command="git config --get remote.origin.url 2>&amp;1" StandardOutputImportance="Low" IgnoreExitCode="true" IgnoreStandardErrorWarningFormat="true" ConsoleToMSBuild="true" Condition="'$(GitRepoUrlRaw)' == ''">
```

Comment:
You should consider using `git ls-remote --get-url` instead which will give you the url for the first remote setup. That at least eliminates your dependency on the name "origin".